anaUltraLatency.py

```diff
@@ -9,10 +9,12 @@
 """
 import numpy as np
 import os
-
-from anaoptions import parser
-from anautilities import make3x8Canvas
 from array import array
+
+import gempython.gemplotting as gemplotting
```

Comment:
Does this PR require a subsequent PR in `cmsgemos`?  I did not see one mentioned in the PR description.  If not where will `gempython.gemplotting` point/be located?

Reply:
It requires a subsequent (not yet made) PR, only for the things it would `import` from there
* This packaging was done such that `cmsgemos`(`gempython.tools`,`gempython.utils`), `vfatqc-python-scripts`(`gempython.vfatqc`), and `gem-plotting-tools` (`gempython.gemplotting`) all create packages/modules within the `gempython` namespace

Actually, this line should have been removed, as the behaviour of this type of alias is useless with the structure (I missed this one apparently)